EventSite/Domain/Model/User.cs

```diff
@@ -81,9 +81,14 @@ public class UserProfile {
             public string Company { get; set; }
             public string Bio { get; set; }
             public string TelephoneNumber { get; set; }
+            public bool IsKeynoteSpeaker { get; set; }
             //public string BlogUrl { get; set; }
             //public string TwitterHandle { get; set; }
             //public bool IsMVP { get; set; }
+            public UserProfile()
+            {
+                IsKeynoteSpeaker = false;
```

Comment:
For the if else I agree but I was getting razor syntax errors there. I'm
going to close this PR and resubmit.
On Mar 21, 2016 8:20 PM, "jlc397 ." <jamescrowley151@gmail.com> wrote:

> Let me resubmit this pull request; I forgot to take taout after I modified
> the document manually in RavenDB. Also, I left commented out lines from you
> or Rob alone because I don't know why they were left commented out.
>
> In EventSite/Domain/Model/User.cs
> <https://github.com/TallyDotNet/event-site/pull/31#discussion_r56919852>:
>
> >              //public string BlogUrl { get; set; }
> >              //public string TwitterHandle { get; set; }
> >              //public bool IsMVP { get; set; }
> > +            public UserProfile()
> > +            {
> > +                IsKeynoteSpeaker = false;
>
> Do we really need this? The default value for the boolean field should be
> false without having to explicitly set it. Also we should drop the
> commented out lines above.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/TallyDotNet/event-site/pull/31/files/035b72e58cf57a6951d259f14316ae8bad84e8a1#r56919852>
>